src/generator/package.integration.js

```diff
@@ -70,63 +70,3 @@ describe('patchman - package using patch-package', function () {
         }]);
     });
 });
-
```

Comment:
I would prefer that we don't remove testing code, with the current implementation we should still be able to use these tests to check that the actual generation process worked, even if we're not reaching out to the patchman API.